src/libcore/pin.rs

```diff
@@ -188,7 +188,8 @@
 //! you do not accidentally use `self`/`this` in a way that is in conflict with pinning.
 //!
 //! Moreover, if your type is `#[repr(packed)]`, the compiler will automatically
-//! move fields around to be able to drop them. As a consequence, you cannot use
+//! move fields around to be able to drop them. In a packed struct, it might even do
```

Comment:
Looking at this again, the "in a packed struct" strikes me as odd... I am going to edit this.